app/pages/Mint/MintBTCTable.tsx

```diff
@@ -0,0 +1,110 @@
+import type { CellProps, Column } from "react-table";
+import { Table } from "~/components/ui/table";
+import { Tooltip } from "~/components/ui/tooltip";
+import { trimAddress } from "~/components/Wallet/walletHelper";
+import { formatBTC } from "~/lib/denoms";
+import type { MintTransaction } from "~/server/Mint/types";
+import { Info } from "lucide-react";
+
+interface MintBTCTableProps {
+	data: MintTransaction[];
+}
+
+function MintTableTooltip({ tooltip, label }: { tooltip: string; label: string }) {
+	return (
+		<Tooltip tooltip={tooltip}>
+			<div className="flex items-center gap-2">
+				{label}
+				<Info size="16" className="text-primary hover:text-orange-400 transition-colors" />
+			</div>
+		</Tooltip>
+	);
+}
+
+const createColumns = (): Column<MintTransaction>[] => [
+	{
+		Header: () => (
+			<MintTableTooltip
+				label="BitCoin Tx ID"
```

Comment:
this was already in previous review, we are re-introducing this again. Its `Bitcoin`